src/test/ui/issues/issue-61187.rs

```diff
@@ -0,0 +1,9 @@
+// edition:2018
```

Comment:
Move this test into the async/await folder (see https://github.com/rust-lang/rust/pull/61138).

Reply:
Done